src/core/weightedGraph.test.js

```diff
@@ -60,4 +61,37 @@ describe("core/weightedGraph", () => {
       expectEqual(wg, wg_);
     });
   });
+
+  describe("overrideWeights", () => {
+    const example = () => {
+      const graph = new Graph().addNode(foo).addNode(bar);
+      const weights = Weights.empty();
+      weights.nodeWeights.set(foo.address, 1);
+      weights.edgeWeights.set(foobar.address, {forwards: 2, backwards: 2});
+      weights.edgeWeights.set(EdgeAddress.empty, {forwards: 2, backwards: 2});
```

Comment:
Nit: think it would be better to have all the weight numbers used be distinct.
To make sure that they didn't for whatever reason get mixed up with another node/edge.

Reply:
Done.